src/services/meteors.service.js

```diff
@@ -7,7 +7,7 @@ export const getMeteorsData = function (request, next) {
   const datesObject = previousWorkWeek(date);
 
   return axios
-    .get(process.env.API_URL, {
+    .get(process.env.METEORS_API_URL, {
```

Comment:
Probably it would be easier to manage if you create config file and then export everything you need there